cmake/Modules/Packages/PYTHON.cmake

```diff
@@ -4,6 +4,7 @@ if(CMAKE_VERSION VERSION_LESS 3.12)
   target_link_libraries(lammps PRIVATE ${PYTHON_LIBRARIES})
 else()
   find_package(Python REQUIRED COMPONENTS Development)
+  target_include_directories(lammps PRIVATE ${Python_INCLUDE_DIRS})
```

Comment:
Why is this needed? Should we included in the target below.

Reply:
I think that might be no longer needed. @lubbersnick can you confirm?

Reply:
My build works without it. It was probably introduced accidentally.